lambda_src/drivers/destination_s3.py

```diff
@@ -31,7 +32,13 @@ def parse_destination_uri(destination: Text) -> Tuple[Text, Text]:
     LOG.debug(f'destination from header is {destination}')
     parsed_url = urlparse(destination)
     bucket = parsed_url.netloc
-    prefix = parsed_url.path.split('/', 2)[1] if parsed_url.path.count('/') >= 2 else ''
```

Comment:
For something like `s3://dev-bucket/foo`, this would return an empty string, since it'd expect at least two `/`s, but the path (`/foo`) would only have one `/`. 

And for a nested path such as this, `s3://dev-bucket/a/b/c/d`, the prefix path returned, would be `a`, since the split would be: `['', 'a', 'b/c/d']`.

Reply:
Ah, I see, so this code only intended for there to be a single folder in `parsed_url.path` instead of a nested URL?

Reply:
Yess, only the first level in the path would be created with the code.